### ISSUE TITLE
after updating or downloading an addon using the nvda addon store change the name of the close button to something like install and close

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#14979]
### Describe the solution you'd like
open the nvda addon store and use the tabs:
Updateable add-ons
or
Available add-ons
update or install an add-on from the list.
At this moment, a novice user is in doubt of what to do, since nvda announces that it has downloaded but in no time informs that the installation will be carried out when pressing the close button.
To solve this, I suggest that when the user installs or updates an add-on that the name of the close button is changed to
install and close.
### Describe alternatives you've considered
the close button after installing an addon may have another name than the one suggested by me.
We may announce to the user after the download is complete that the installation will take place after closing the addon store window.
We can only document this very well in the help, this is the least desired solution for me.
### Additional context
I'm thinking of new or inexperienced users in using nvda

**Comment by XLTechie** [comment]
1. While updating is new, installing, removing, or changing enablement has 
always required closing the add-ons manager in previous versions of NVDA.
Yet you did not raise an issue asking that the close button of the add-ons 
manager be changed thus.
(I'm sorry if you did but I missed it--I have never heard of any such issue.)

Why does the store need this, but the manager did not?

2. If someone updates one add-on, installs a second one, removes a third, and 
disables a fourth: what should the close button be called then?

**Comment by fernando-jose-silva** [comment]
I decided to make this request, when working with this new skin, to imagine my students who have a lot of difficulty using this screen in the future.
from that point of view, yes, this change should have existed since the days of the plugin manager.
In the case that the user performs several activities, the button could be called apply changes.
But that button, that name, doesn't fill the gap I noticed when realizing that nvda downloaded the addon AND didn't install it, and still didn't inform the user that he would need to press the close button for the addon to be there, this is not Of course, I've finished downloading the add-on, if I close the window, at least in my way of thinking, it would cause the action that downloads the add-on to be cancelled.
